### PR TITLE
Chore/update integration example

### PR DESCRIPTION
**Pull request by vinzlange**
tested and updated integration example with v5 and app 4.0.1

### CODE REVIEW

File: _docs_integrate/integration-example.md [hunk 1]
```diff
@@ -79,12 +77,9 @@ To do so, execute `POST /api/v2/Attributes` with the following payload:
 }
 ```
 
-You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:pilot.enmeshed.eu:dids:9364c879fcc41c87c6ecbd`.
-{: .notice--info}
-
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:84]: why did you remove this? This is still perfectly valid.

File: _docs_integrate/integration-example.md [hunk 2]
```diff
@@ -162,8 +157,8 @@ Even though the Requests are validated during the RelationshipTemplate creation,
 
 ### Connector: Create a RelationshipTemplate
 
-If the Response is successful, we can create the RelationshipTemplate.
-To do so, we use the `content` we just validated.
+If the response successfully confirms all attributes, we can create the RelationshipTemplate.
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:160]:
```suggestion
If the response is successful, we can create the RelationshipTemplate.
```

it sounds like this would only validate attributes, but it does way more things.

File: _docs_integrate/integration-example.md [hunk 3]
```diff
@@ -193,24 +188,11 @@ For this, execute the `GET /api/v2/RelationshipTemplates/{id}` route (Accept Hea
...
-  - select the existing profile
-  - navigate to "Contacts"
-  - click on "Add contact"
-  - hold the camera in front of the QR code
+Scanning the qr code should result in a screen similar to the one below, where you can see the information that you added as `content` to the RelationshipTemplate.
 
-All three paths should result in a screen similar to the one below, where you can see the information that you added as `content` to the RelationshipTemplate.
-
-!["Add contact" screen]( {{ '/assets/images/add-contact-screen.jpg' | relative_url }} )
+![Add contact screen](/assets/images/add-contact-screen.jpg){: width="40%"}
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:195]: you forgot the `| relative_url`

File: _docs_integrate/integration-example.md [hunk 4]
```diff
@@ -290,6 +272,11 @@ Then, tap on "New Message", enter a subject and body and tap on "Send".
 In order to fetch the Message, we need to synchronize the Connector with the Backbone again.
 
 {% include rapidoc api_route_regex="^post /api/v2/Account/Sync$" %}
+
+After syncing, all messages can be displayed with the `GET /api/v2/Messages` route.
+
+{% include rapidoc api_route_regex="^post /api/v2/Messages$" %}
```
Comment (Magnus-Kuhn) [inline _docs_integrate/integration-example.md:278]:
```suggestion
{% include rapidoc api_route_regex="^get /api/v2/Messages$" %}
```

File: _docs_integrate/integration-example.md [hunk 5]
```diff
@@ -193,24 +188,11 @@ For this, execute the `GET /api/v2/RelationshipTemplates/{id}` route (Accept Hea
...
-- If you want to use a new profile:
-  - click on the "+ New profile" button
-  - click on "Scan code"
-  - hold the camera in front of the QR code
-- If you want to use an existing profile:
-  - select the existing profile
-  - navigate to "Contacts"
-  - click on "Add contact"
-  - hold the camera in front of the QR code
+Scanning the qr code should result in a screen similar to the one below, where you can see the information that you added as `content` to the RelationshipTemplate.
```
Comment (Magnus-Kuhn) [inline _docs_integrate/integration-example.md:193]: I think we capitalize QR

**Comment by Magnus-Kuhn** [comment]
https://js-soft.atlassian.net/wiki/spaces/JSSNMSHDD/pages/3893526531/Style+guide as an explanation for the capitalization comments @vinzlange

File: _docs_integrate/integration-example.md (hunk 1 above)
Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: The Connector address is no longer required for creating an attribute, so why should it be mentioned in the tutorial? Limit the example to the most necessary to minimize complexity.

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: image

?

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: @britsta is the right side in the screenshot even correct?

File: _docs_integrate/integration-example.md (hunk 2 above)
Comment (vinzlange) [reply _docs_integrate/integration-example.md:160]: how else could you write it. my problem is that the response comes back successfully (201), even if there are problems in it. I would like to make that clearer

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:160]: wtb `if the Connector states your Request as valid`?

File: _docs_integrate/integration-example.md (hunk 1 above)
Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: also - do we really don't need the address in the whole page?

File: _docs_integrate/integration-example.md [hunk 6]
```diff
@@ -79,12 +77,9 @@ To do so, execute `POST /api/v2/Attributes` with the following payload:
 }
 ```
 
-You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:pilot.enmeshed.eu:dids:9364c879fcc41c87c6ecbd`.
-{: .notice--info}
-
 {% include rapidoc api_route_regex="^post /api/v2/Attributes$" %}
 
-{% include copy-notice description="Save the `id` of the Attribute that you can find in the Response. You will need it in the next step." %}
+{% include copy-notice description="Save the `id` and the `owner` of the Attribute that you can find in the Response. You will need it in the next step." %}
```
[the remark this answers is not in the document]
Comment (vinzlange) [reply _docs_integrate/integration-example.md:82]: @Magnus-Kuhn response always in lowercase? is often capitalized in the file

File: _docs_integrate/integration-example.md (hunk 1 above)
Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: before you only needed it when creating the attribute, since the update I need it in your screenshot above, if that's what you want and not a bug (it can no longer be “”)

File: _docs_integrate/integration-example.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jkoenig134) [reply _docs_integrate/integration-example.md:82]: it depends, if we mean a http response then small, if we mean an enmeshed Response, then capital

File: _docs_integrate/integration-example.md (hunk 1 above)
Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: but you need it, so you should re-add the information on how to get to the information..

Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: in line 82: the response from creating the attribute gives you the address

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: but how should I get the connection that <your-connector-address> is the owner you copied before... makes no sense for me.

Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: why do I now need the owner for the request and can no longer “”, is that intentional? 
And I get the id and the owner when creating the attribute, why can't I use both of them.

Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: I get: "error.consumption.requests.invalidRequestItem: 'The provided IdentityAttribute belongs to someone else. You can only share own IdentityAttributes.' at 2024-08-23T14:00:11.876Z",
I think this is a bug. If the bug is fixed the owner can be “” and then the owner of the IdentityInfo is not needed anywhere

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: @britsta wdym?

Comment (britsta) [reply _docs_integrate/integration-example.md:84]: It is intended that an empty string can no longer be specified here as the owner, for several reasons:
- The empty string is often used as a placeholder for other RequestItems if the recipient of the Request is not yet known (i.e. if the Request is sent via a RelationshipTemplate and not a Message). To avoid confusion, the empty string should mainly be used for this purpose and no longer as a way of saving typing work for an already know address (e.g. my address).
- With a ShareAttributeRequestItem, we can share an IdentityAttribute that already exists in the database, in which the address of the `owner` is explicitly specified. Normally, it should be copied from the database and can then be inserted in the `attribute` property of the ShareAttributeRequestItem in exactly this form (with specified `owner` and not empty string).
- In the past, it was possible with some RequestItems, if you wanted to cause a bug, that Attributes could be stored in the database that actually had an empty string as `owner`. This should hopefully be fixed by now, but has led us to become stricter here and only want to use an empty string if we have no choice, i.e. do not know an address.

Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: Then the error message I get when validating is incorrect or at least misleading.
But I still see no reason to use: `/api/v2/Account/IdentityInfo`

**Comment by britsta** [comment]
> https://js-soft.atlassian.net/wiki/spaces/JSSNMSHDD/pages/3893526531/Style+guide as an explanation for the capitalization comments @vinzlange

Thanks for spreading the style guide @Magnus-Kuhn ! :D 

(Btw, as far as I know, we should avoid posting internal links on GitHub.)

File: _docs_integrate/integration-example.md (hunk 1 above)
Comment (britsta) [reply _docs_integrate/integration-example.md:84]: I would actually have expected a different error message as well and also found the problem in the code, will be fixed soon.

Comment (britsta) [reply _docs_integrate/integration-example.md:84]: The PR https://github.com/nmshd/runtime/pull/250 has just been merged so that an error message is now displayed that makes more sense.

When creating an IdentityAttribute, the address of the `owner` can be found in the success response, so that you do not necessarily have to use the use case to query the currently used Identity to get the address. I would probably mention it in a `{: .notice--info}` box anyway, because I think it would be beneficial for the reader.

Comment (vinzlange) [reply _docs_integrate/integration-example.md:84]: I am very dissatisfied, if it says save the id and the user from the response and in the note directly below it says oh if you forgot to save the user you can also do that with the use-case.

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:84]: true, but if you should have copy pasted this why is there a placeholder?

File: _docs_integrate/integration-example.md [hunk 7]
```diff
@@ -108,7 +103,7 @@ Let's assume the Connector needs to know the given name and surname of its conta
             "mustBeAccepted": true,
             "attribute": {
               "@type": "IdentityAttribute",
-              "owner": "",
+              "owner": "<the owner of the Attribute created above>",
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:106]:
```
              "owner": "<the `owner` of the Attribute created above>",
```
or
```
              "owner": "<the owner propery of the Attribute created above>",
```
I like the second more :)

File: _docs_integrate/integration-example.md [hunk 8]
```diff
@@ -290,7 +272,12 @@ Then, tap on "New Message", enter a subject and body and tap on "Send".
 In order to fetch the Message, we need to synchronize the Connector with the Backbone again.
 
 {% include rapidoc api_route_regex="^post /api/v2/Account/Sync$" %}
-The Response should contain a Message with the `content` you entered in the App.
+
+After syncing, all Messages can be displayed with the `GET /api/v2/Messages` route.
+
+{% include rapidoc api_route_regex="^get /api/v2/Messages$" %}
+
+The response should contain a Message with the `content` you entered in the App.
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:280]: we should also talk about that there will be an event that you can subscribe to. Take the entry to events as soon and as often as possible

File: _docs_integrate/integration-example.md (hunk 7 above)
Comment (vinzlange) [reply _docs_integrate/integration-example.md:106]: you mean property?

Comment (jkoenig134) [reply _docs_integrate/integration-example.md:106]: yep

**Review by jkoenig134 — APPROVED** [review]
